Update README.md

mariusbutuc opened the pull request:
Advertise the latest release

mariusbutuc committed:
Update README.md

Advertise the latest release

obrok commented on README.md:
````diff
@@ -80,17 +80,21 @@ iex> get_and_update_in([1, 2, 3], [Lens.all() |> Lens.filter(&Integer.is_odd/1)]
         end
+        ```
 
   2. Ensure lens is started before your application:
````
Actually this section is kinda obsolete, now that mix automatically bundles deps with the application and since lens doesn't include any workers - would you like to clean this up?

obrok reviewed (commented):
Thanks for the PR! I totally forgot this section existed. Maybe you'd also like to move it to the top of the README? Seems kinda important.

mariusbutuc committed:
Move the README Installation section to the top

Remove the obsolete step suggesting to ensure lens is started before the 
application. 

It is obsolete since mix automatically bundles deps with the application 
and since lens doesn't include any workers.

mariusbutuc commented:
Thanks for the follow-up and the detailed suggestions @obrok; integrated them all. 👍

obrok reviewed (approved):
❤️
